build.rs

```diff
@@ -39,7 +39,21 @@ fn find_leptonica_system_lib() -> Option<String> {
     Some(include_path.to_string_lossy().to_string())
 }
 
-#[cfg(all(not(windows), not(target_os = "macos")))]
+#[cfg(target_os = "linux")]
```

Comment:
Is this the same as the MacOS pkg-config search?

Can we delete one of them and use this `cfg` line?

```suggestion
#[cfg(any(target_os = "linux", target_os = "linux"))]
```